plugins/salesforce-commerce-api-plugin/src/api.ts

```diff
@@ -93,7 +93,7 @@ export class Api {
   async validateConfig(){
        const response = await this.request(`validate-config`);
        if(response.errors){
-        throw response.errors[0]?.title;
+        throw 'We failed to authenticate your access to salesforce commerce api. Please review all plugin fields and make sure they are correct.';
```

Comment:
sweet. Can you make sure that the spelling is exactly correct? From looking it up here: https://github.com/SalesforceCommerceCloud/commerce-sdk 

it looks like the exact name is "Salesforce Commerce Cloud B2C Commerce API", but you probably know best.

Reply:
Done